source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRShowCardTarget.mm

```diff
@@ -20,7 +20,8 @@ @implementation ACRShowCardTarget
     ACOHostConfig *_config;
     __weak UIView<ACRIContentHoldingView> *_superview;
     __weak UIViewController *_vc;
-    __weak UIView *_adcView;
+    UIView *_adcView;
```

Comment:
Is there a reason that we changed __weak to strong?

Reply:
For some reason,the _adcView reference was lost, changing to stron pointer keeps the reference alive

---
In reply to: [174966199](https://github.com/Microsoft/AdaptiveCards/pull/1259#discussion_r174966199) [](ancestors = 174966199)

Reply:
it's because of ARC, and LN130 has been removed. the super view used to have reference to _adcView. it's better keep __weak. we should keep LN130. and make the view hidden. _adcView.hidden = YES after or before adding _adcView to super view. Then remove __justCreated variable by checking hidden property.

---
In reply to: [175234864](https://github.com/Microsoft/AdaptiveCards/pull/1259#discussion_r175234864) [](ancestors = 175234864,174966199)